Tests/Schema.NET.Test/JobPostingTest.cs

```diff
@@ -10,7 +10,7 @@ public class JobPostingTest
         private readonly JobPosting jobPosting = new JobPosting()
         {
             Title = "Fitter and Turner", // Required
-            Description = "<p>Widget assembly role for pressing wheel assemblies.</p><p><strong>Educational Requirements:</strong> Completed level 2 ISTA Machinist Apprenticeship.</p><p><strong>Required Experience:</strong> At least 3 years in a machinist role.</p>", // Required
+            Description = "Widget assembly role for pressing wheel assemblies. Educational Requirements: Completed level 2 ISTA Machinist Apprenticeship. Required Experience: At least 3 years in a machinist role.", // Required
```

Comment:
Any reason to remove the HTML tags here?

Reply:
Originally did this as my first changes had made the default `.ToString()` use the HTML escaping. As such I would've had to encode these tags. I think encoding would work, but I couldn't find any examples on schema.net of html tags in or being allowed in these properties, so I just removed. Should be able to add back though if you'd like and we could test the encoding with the Google schema tester tool.

Reply:
I think I remember copying this data from a Google website, probably the schema testing tool. It would be nice to match up with it to make sure everything is working with Google, since that is the main use case for this. Very minor change, then we can merge and release.